deployment/data-feeds/changeset/set_feed_admin.go

```diff
@@ -14,11 +14,18 @@ import (
 var SetFeedAdminChangeset = deployment.CreateChangeSet(setFeedAdminLogic, setFeedAdminPrecondition)
 
 func setFeedAdminLogic(env deployment.Environment, c types.SetFeedAdminConfig) (deployment.ChangesetOutput, error) {
-	state, _ := LoadOnchainState(env)
+	state, stateErr := LoadOnchainState(env)
```

Comment:
The reason why we don't handle the error here is because we handle it in the `setFeedAdminPrecondition` function in the same file which runs before the actual logic. Same applies to other errors

Reply:
well, if the contract isn't there or chain isn't there I was getting NPEs

Reply:
that's weird. If you run the `setFeedAdminLogic` directly then yes, but in context of CLD migration runs, the precondition function runs first. I will double check if the error is handled there or not

Reply:
I am not using migrations. I call this: ` df_changeset.SetFeedAdminChangeset.Apply`. Is not that a correct entry point?

Reply:
If you call `Apply` directly then yes, the precondition function won't work, however in CLD there is a [wrapper](https://github.com/smartcontractkit/chainlink-deployments/blob/main/pkg/migrations/common.go#L126)  that calls the precondition function first, which does all the checks. Therefor we don't have these checks in the function logic itself

Reply:
I am not using `chainlink-deployments`. We shouldn't assume that all users will. That repo is only for public networks deployments. What about all other use cases?